pkgs/development/libraries/givaro/default.nix

```
@@ -31,6 +31,16 @@ stdenv.mkDerivation rec {
       url = "https://github.com/linbox-team/givaro/commit/a81d44b3b57c275bcb04ab00db79be02561deaa2.patch";
       hash = "sha256-sSk+VWffoEjZRTJcHRISLHPyW6yuvI1u8knBOfxNUIE=";
     })
+    (fetchpatch {
```

```suggestion
    # https://github.com/linbox-team/givaro/issues/226
    (fetchpatch {
```

The gcc14 patch is a lot longer than just the required fix. This is OK, but it's worth including a link to the issue (as proposed) for ease of verification.

---

thanks -- that was sloppy. I'll trim the patch to only include the fix.